src/AddExperience.js

```diff
@@ -0,0 +1,88 @@
+// import { format } from 'prettier';
+import React, { useState } from "react";
+import "./App.css";
+
+export default function AddExperience({ userId, setUserId }) {
+  const [inputs, setInputs] = useState({ name: "", proficiency: "", logo: "" });
+  const [alert, setAlert] = useState("");
+
+  const handleChange = (event) => {
+    const { name, value } = event.target; // Destructure name and value from the event target
+    setInputs((prevInputs) => ({
+      ...prevInputs, // Spread existing input values
+      [name]: value, // Update the specific input field by its name
+    }));
+  };
+  const updateList = (event) => {
+    event.preventDefault(); //stop page refresh
+    console.log("update to data base==========", inputs);
+    // alert("experience saved!")
+
+    const url = "http://localhost:5000/resume/skill";
```

Comment:
are you adding Experience or Skill?

in either case, it needs to show me what I added:

https://github.com/user-attachments/assets/0fdd0e26-b2b1-496d-8139-373e79286a0a